.github/workflows/ci.yml

```diff
@@ -49,13 +49,13 @@ jobs:
         run: yarn --frozen-lockfile
 
       - name: Build packages
-        run: yarn build
+        run: yarn build --filter=[main...${{GITHUB_REF_NAME}}]
```

Comment:
This brilliant! Looks like there is an [error with accessing the env](https://github.com/Shopify/polaris/actions/runs/4080572128/workflow#L52) variable here though.

When using `${{ ... }}` I think you'd access it on the `env` object:

```
run: yarn build --filter=[main...${{env.GITHUB_REF_NAME}}]
```

Or I think you could also skip the curly braces and just do

```
run: yarn build --filter=[main...$GITHUB_REF_NAME]
```

Reply:
Oooh, does this syntax filter for only workspaces which have changes since the last commit to `main`?

What about the site where we want to re-build the docs site even if there's been no changes to the site, but there _have_ been changes to the components which live in a different workspace but are linked in to the site?

Reply:
We can always do that but for CI I would hope we are only testing code that changes. If it does break I would be happy to revert.